README.md

```diff
@@ -652,6 +653,42 @@ Creates a version of the function that will only be run after first being called
  --- | --- | --- | --- | --- |
    ✔  |  ✔ |  ✔ |  ✔ |  ✔  |  
  
+ ## _.assign
+The method is used to copy the values of all enumerable own properties from one or more source objects to a target object.
+
+  ```js
+  // Underscore/Lodash
```

Comment:
Hi, Underscore has _.assign method.

See http://underscorejs.org/#extendOwn

>extendOwn_.extendOwn(destination, *sources) Alias: assign 

You can type `_.assign` in the console of the page, it linked a underscore library.